openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -67,64 +105,160 @@ def load_casting(project_name, shot_name):
     gazu.client.set_host(os.environ["KITSU_SERVER"])
     gazu.log_in(os.environ["KITSU_LOGIN"], os.environ["KITSU_PWD"])
 
-    episode_name, sequence_name, shot_name = shot_name.split("_")
+    shot_data = get_asset_by_name(project_name, shot_name, fields=["data"])
```

Comment:
if you're fetching the data, then:
```suggestion
    shot_data = get_asset_by_name(project_name, shot_name, fields=["data"])["data"]
```
because shot_data["data"] makes no sense.